simulation.py

```diff
@@ -26,7 +27,7 @@ def update_ip_lat_long_map(ip_lat_long_map_file):
 if __name__ == '__main__':
   parser = argparse.ArgumentParser()
   parser.add_argument('--disable-concurrency', action='store_false', help='disable concurrency (no delays on requests)')
-  parser.add_argument('--algorithm', choices=['volley', 'greedy', 'distributed'], help='the algorithm used for replication')
+  parser.add_argument('--algorithm', choices=['volley', 'greedy', 'simple_centralized_greedy',  'distributed'], help='the algorithm used for replication')
```

Comment:
may be change `simple_centralized_greedy` to something shorter? it's gonna be a pain to type this haha.